feat(workspace/warehouse_app): add resource to manage warehouse application (#5856)
huaweicloud/terraform-provider-huaweicloud: docs/resources/workspace_app_warehouse_app.md

---
subcategory: "Workspace"
layout: "huaweicloud"
page_title: "HuaweiCloud: huaweicloud_workspace_app_warehouse_app"
description: |-
  Use this resource to add a application to Workspace APP warehouse within HuaweiCloud.
---

# huaweicloud_workspace_app_warehouse_app

> Use this resource to add a application to Workspace APP warehouse within HuaweiCloud.

Reviewer: ```suggestion
Manages an application resource of Workspace APP warehouse within HuaweiCloud.
```
